app/src/main/java/com/example/the_dagger/learnit/utility/LogUtils.java

```diff
@@ -0,0 +1,107 @@
+package com.example.the_dagger.learnit.utility;
```

Comment:
I can't seem to see the use for this LogUtils, we anyways to enter both the Tag and the message while logging anything.
A good idea would be to modify the utils method to automatically insert the Tag, asking us to just insert the log message.